test/DynamoCoreWpfTests/GalleryTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.IO;
+using Dynamo;
+using Dynamo.Wpf.ViewModels.Core;
+using NUnit.Framework;
+
+namespace DynamoCoreWpfTests
+{
+    [TestFixture]
+    public class GalleryTests : UnitTestBase
+    {
+        [Test]
+        public void LoadGalleryContents()
+        {
+            //Test loading of xml gallery contents
+            var galleryFilePath = Path.Combine(Directory.GetCurrentDirectory(),
+                @"..\..\..\test\core\gallery\NormalGalleryContents.xml");
```

Comment:
`GetCurrentDirectory` is highly dependent on the previous caller, so it can really be anything. Let's use `GetExecutingAssembly` instead.